Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM rust:1-slim-stretch as builder
+
+WORKDIR /srv/addresses-importer
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y gcc-4.8 && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+ENV CC "gcc-4.8"
+RUN git clone https://github.com/openvenues/libpostal \
+    && cd postal \
+    && ./bootstrap.sh \
+    && ./configure --datadir=$(pwd)/data \
+    && make install \
```

Comment:
I didn't know that `make install` triggers a build :)

Maybe we still want to explicitly run a `make -jX` (X = 4?) to speed things up?